docs/application/dotnet/getting-started/overview.md

```diff
@@ -0,0 +1,7 @@
+# Getting Started with .NET Applications
+
+When creating your first Tizen .NET application, you can select from the following options, which all use the C# language:
+
+-   [.NET application](first-app.md)
+
+    .NET application is targeted to run on smart phones, smart watches and TV can take advantage of a wide variety of functionalities available on those devices.
```

Comment:
Suggestion:
.NET application is targeted to run on smart phones, smart watches, and TV. The applications can take advantage of a wide variety of functionalities available on those devices.